### DISCUSSION TITLE
remove the wineskin app on the applications by default

### DISCUSSION THREAD

**Discussion by ghost** [The-Wineskin-Project/WineskinServer#264] [General]
please remove that feature cause if i delete the wineskin app it stays there
![Screen Shot 2022-05-24 at 7 37 11 PM](https://user-images.githubusercontent.com/82131956/170167794-84aa61aa-eb0f-4d90-ab98-6c856d9de98a.png)
 
and i put the application on a folder but the application is set to the application

**Comment by Gcenx** [comment]
I’ve read this multiple times now and I’m not able to understand your question

**Comment by emendelson** [comment]
I think the question **may** have something to do with the wrapper's icon, and with what happens when you set a custom icon: macOS may not change the displayed icon until you move the wrapper to a different folder. (This was discussed in another thread.) But this is only a wild guess. It's the most puzzling post I've ever seen on GitHub.

**Reply by Gcenx** [reply]
It’s rather confusing indeed as the screenshot seems to be from spotlight after an app is removed as it’s cache usually isn’t updated so shows the icon with a question mark, this can happen in any part of Finder too.

My guess is the message was translated from there native language to English.
